godtools/App/Features/ToolsFilter/Presentation/ToolFilterLanguageSelectionView/ToolFilterLanguageSelectionViewModel.swift

```diff
@@ -43,22 +45,37 @@ class ToolFilterLanguageSelectionViewModel: ObservableObject {
         self.selectedLanguage = languageFilterSelectionPublisher.value
         self.flowDelegate = flowDelegate
         
+        getCurrentAppLanguageUseCase
+            .getLanguagePublisher()
+            .assign(to: &$appLanguage)
+        
+        $appLanguage.eraseToAnyPublisher()
+            .flatMap { appLanguage in
+                
+                return viewToolFilterLanguagesUseCase.viewPublisher(filteredByCategoryId: selectedCategory.id, translatedInAppLanguage: appLanguage)
+            }
+            .receive(on: DispatchQueue.main)
+            .sink { viewLanguageFiltersDomainModel in
+                
+                self.allLanguages = viewLanguageFiltersDomainModel.languageFilters
```

Comment:
Capturing self here will cause a retain cycle.  Will need to use capture list [weak self].